src/main/java/bio/terra/service/auth/iam/IamAction.java

```diff
@@ -50,7 +57,7 @@ public String toString() {
   @JsonCreator
   public static IamAction fromValue(String text) {
     for (IamAction b : IamAction.values()) {
-      if (b.name().equals(StringUtils.upperCase(text))) {
+      if (b.samActionName.equalsIgnoreCase(text)) {
```

Comment:
That's true.  My reasoning with this change is that it more directly reflects that we only care about a case-insensitive match rather than having to convert one side or the other, but that may boil down to personal preference.